backend/molgenis-emx2/src/main/java/org/molgenis/emx2/Row.java

```diff
@@ -358,6 +358,9 @@ public boolean notNull(String columnName) {
   }
 
   public boolean isNull(String columnName, ColumnType type) {
+    if (type.isArray()) {
+      return get(columnName, type) == null || ((Object[]) get(columnName, type)).length == 0;
```

Comment:
"get"  What ? getColumnValue ?? , why would i need to supply the type 
the method is called isNull , but checks if null or empty array, and i looks like 'get' returns null in case of array but its hard to see what is going in due to the use of single char variable names

Reply:
? single var variable names?

Reply:
N.B. the row cannot know what type you want to get() the data. So you either can use typed getters, e.g. getString, getInt etc, or you must parameterize your request. For example, if you are currently parsing a CSV file then the underlying value might a comma seperated string that first needs parsing before you can assess nullable. Now, if we think hard about all edge cases we might be able to asses 'nullOrEmpty'-ness based on the underlying this.values.get(columnName) which might be null, "", [] without using columnType.

Basically I propose to rename to 'isNullOrEmpty' to clarify there is a bit more to this.